introduction.md

```diff
@@ -5,3 +5,10 @@ Again, updating the file to proceed in the CodeSchool Mastering GitHub.
 And again I'm updating the file in an attempt to complete 1.6
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?